src/main/java/com/nongfenqi/nexus/plugin/rundeck/RundeckMavenResource.java

```diff
@@ -114,11 +123,13 @@ public Response content(
             return commitAndReturn(NOT_FOUND, tx);
         }
 
+        String folderVersion = version.replaceAll("-\\d{8}\\.\\d{6}\\-\\d+", "-SNAPSHOT");
```

Comment:
if version == null ?

Reply:
I don't believe that version can be null? or can be?

There was a problem with a folder naming for snapshot:
http://nexus.local/repository/snapshots/net/[companyname]/core-api/4.3.309-SNAPSHOT/core-api-4.3.309-20210120.055711-1.jar

so i needed to replace timestamp with SNAPSHOT to get it working with latest nexus.